app/manage/pods/rules-form/conditional-form-directive/index.js

```diff
@@ -1,6 +1,20 @@
 'use strict';
 var _ = require('lodash');
 
+var sensorOptions = [{id: 'temp', displayName: 'Temperature'},
```

Comment:
The whole rule engine will be replaced soon.

That's why we have chosen the shortest and easiest way.